.github/workflows/test-workflow.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   build:
     strategy:
       matrix:
-        java: [13]
+        java: [14]
```

Comment:
Need to update in other 2 Github actions as well

Reply:
Good catch. Done